whisper_online.py

```diff
@@ -430,10 +430,10 @@ def split(self, text):
                 return tokenize_uk.tokenize_sents(text)
         return UkrainianTokenizer()
 
-    # supported by fast-mosestokenizer
+    # supported by mosestokenizer (1.2.1)
     if lan in "as bn ca cs de el en es et fi fr ga gu hi hu is it kn lt lv ml mni mr nl or pa pl pt ro ru sk sl sv ta te yue zh".split():
-        from mosestokenizer import MosesTokenizer
-        return MosesTokenizer(lan)
+        from mosestokenizer import MosesSentenceSplitter
```

Comment:
I need this to work in both options, on Unix as well as your Apple.

Please, put here "if MosesTokenizer does not have split method, then `from mosestokenizer import MosesSentenceSplitter`, and create an object that calls it in a split function, analogically to UkrainianTokenizer.